test/controller.test.ts

```diff
@@ -0,0 +1,143 @@
+import { describe, test, afterEach, before } from "mocha";
+import { spy, restore, assert, match } from "sinon";
+import yargs from "yargs/yargs";
+import fetch, { Headers, Request, Response } from "node-fetch";
+import { getAccounts } from "@tableland/local";
+import * as mod from "../src/commands/controller.js";
+
+describe("commands/controller", function () {
+  this.timeout("30s");
+
+  before(async function () {
+    if (!globalThis.fetch) {
```

Comment:
Mocha has a`--file` flag that will let us indicate a global setup and teardown file.  I _think_ that would work instead of doing this in multiple test files.
example:
https://github.com/tablelandnetwork/local-tableland/blob/joe/pkgdval/test/mocha.opts
https://github.com/tablelandnetwork/local-tableland/blob/joe/pkgdval/test/setup.mjs

With that said, if we are ok running tests on node >= 16 we won't need the fetch polyfil anymore.